vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp

```diff
@@ -312,30 +323,29 @@ winrt::fire_and_forget ReactImage::SetBackground(bool fireLoadEndEvent) {
       }
 
       if (source.sourceFormat == ImageSourceFormat::Svg) {
-        winrt::SvgImageSource svgImageSource{imageBrush.ImageSource().try_as<winrt::SvgImageSource>()};
+        winrt::SvgImageSource svgImageSource{};
 
-        if (!svgImageSource) {
-          svgImageSource = winrt::SvgImageSource{};
+        strong_this->m_svgImageSourceOpenedRevoker =
+            svgImageSource.Opened(winrt::auto_revoke, [weak_this, fireLoadEndEvent](const auto &, const auto &) {
```

Comment:
This is documented [here](https://learn.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/weak-references#if-you-use-a-member-function-as-a-delegate), use this form `event_source.Event({ get_weak(), &EventRecipient::OnEvent });`

Reply:
This isn't new code. Showing up as new because I'm removing the if statement around it. Keeping the change scoped to the fix because we need to backport it.